src/Uno.Sdk/targets/Uno.Common.Wasm.targets

```diff
@@ -83,13 +83,33 @@
 			<!-- Remove the files Web SDK included files. This will also remove duplicates, regardless of the metadata -->
 			<Content Remove="@(_ContentToUpdate)"
 					 Condition="'%(_ContentToUpdate.IsWebSdk)'=='true' AND '%(_ContentToUpdate.IsDefaultItem)'!='true'" />
-			
+
 			<!-- Add back the non-websdk content -->
 			<Content Include="@(_NonWebSdkContent)" KeepDuplicates="false" />
 
 			<!-- Cleanup -->
 			<_ContentToUpdate Remove="@(_ContentToUpdate)"/>
 			<_NonWebSdkContent Remove="@(_NonWebSdkContent)"/>
+
+		</ItemGroup>
+	</Target>
+
+	<Target Name="_UnoAdjustWwwrootContent"
+			BeforeTargets="AssignTargetPaths">
+
+		<ItemGroup>
+			<_wwwRootContentToUpdate Include="$(MSBuildProjectDirectory)/$(WasmProjectFolder)/wwwroot/**" />
+
+			<Content Remove="@(_wwwRootContentToUpdate)" />
+
+			<Content Include="@(_wwwRootContentToUpdate)">
+				<Link>%(RecursiveDir)%(Filename)%(Extension)</Link>
+				<UnoDeploy>Root</UnoDeploy>
+				<!-- net9+ compatibility -->
+				<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+			</Content>
+
+			<_wwwRootContentToUpdate Remove="@(_wwwRootContentToUpdate)" />
```

Comment:
Originally, do we have every single file in `wwwroot` included in `Content`?

If `wwwroot` originally had `FileA` and `FileB`, but only `FileA` was included in `Content`. With these changes, we'll be adding both `FileA` and `FileB` as Content. Not sure if it's intended.

Reply:
`wwwroot` is implicitly including everything, the same way it is with aspnet core apps. It got broken when the task above was introduced and we missed regression tests for that.

Reply:
My comment means the other way around, does `Content` include everything in `wwwroot`? (not that wwwroot contains everything from Content)

Reply:
Also, how things are going with non-Uno.Sdk?

Reply:
Duplicates in this context are handled properly by the bootstrapper. Without the uno.sdk is different, it's managed by the bootstrapper itself.